api/provider/aws/environment_create.go

```diff
@@ -83,10 +95,10 @@ func (e *EnvironmentProvider) Create(req models.CreateEnvironmentRequest) (*mode
 	return e.Read(environmentID)
 }
 
-func (e *EnvironmentProvider) createSG(environmentID, vpcID string) (*ec2.SecurityGroup, error) {
+func (e *EnvironmentProvider) createSG(groupName, description, vpcID string) (*ec2.SecurityGroup, error) {
 	input := &ec2.CreateSecurityGroupInput{}
-	input.SetGroupName(environmentID)
-	input.SetDescription(fmt.Sprintf("SG for Layer0 environment %s", environmentID))
+	input.SetGroupName(groupName)
```

Comment:
Question: Why do we not initialize the struct inline rather than calling setters? Is it just a styling convention?